sdk/ai/azure-ai-projects/samples/agents/async_samples/sample_agents_vector_store_batch_enterprise_file_search_async.py

```diff
@@ -41,15 +45,24 @@ async def main():
 
         # We will upload the local file to Azure and will use it for vector store creation.
         _, asset_uri = project_client.upload_file("../product_info_1.md")
-        ds = VectorStoreDataSource(asset_identifier=asset_uri, asset_type=VectorStoreDataSourceAssetType.URI_ASSET)
-        vector_store = await project_client.agents.create_vector_store_and_poll(file_ids=[], name="sample_vector_store")
+        ds = VectorStoreDataSource(
+            asset_identifier=asset_uri,
+            asset_type=VectorStoreDataSourceAssetType.URI_ASSET,
+        )
+        vector_store = await project_client.agents.create_vector_store_and_poll(
```

Comment:
file_ids is optional, so I thin you don't need to type file_ids[]

Reply:
The problem is that the default value is None and that results in REST request, missing file_ids, resulting in 400 response:
```
Invalid parameters for create_vector_store_and_poll. Please provide either 'body', 'file_ids', 'store_configuration', or 'name' and 'expires_after'.
```